test/unit/math/prim/err/check_positive_test.cpp

```diff
@@ -3,10 +3,10 @@
 #include <limits>
 #include <vector>
 
-TEST(ErrorHandlingArr, CheckPositive) {
-  using stan::math::check_positive;
-  const char* function = "check_positive";
+using stan::math::check_positive;
```

Comment:
I think after #1947 we're not having any global statements in the tests

Reply:
If possible yes, that would be a great practice to follow.